client/src/styles/_header.scss

```diff
@@ -43,39 +55,54 @@
     }
   }
 
-  li > a {
-    color: unset !important;
+  .menu-item {
+    color: unset;
     display: inline-block;
-    padding: 0 1.5rem;
+    padding: 0 15px;
     width: 100%;
 
     &.active {
       color: $primary-color !important;
     }
 
-    @media (min-width: $breakpoint-tablet) {
-      border-bottom: 2px solid transparent;
-      transition: border-color 0.3s cubic-bezier(0.645, 0.045, 0.355, 1);
-      &:hover {
-        border-bottom: 2px solid $primary-color;
-      }
-      &.active {
-        border-bottom: 2px solid $primary-color;
-      }
+    @media (max-width: $breakpoint-desktop) {
+      padding: 0 0.8rem;
     }
   }
+
+  a.add-building {
+    color: #fff;
+    padding-top: 0px;
+  }
 }
 
 .ant-layout-header {
+  position: absolute;
```

Comment:
i would like to keep the `position: relative` here and remove the margin-top from `.ant-layout{`.
There is any reason for using `position: absolute` and not relative?

Reply:
Well the only reason for using `position: absolute` was to have header span the full width of the screen while the rest of the elements contained by `.ant-layout` would span only 70%, 85% or 100% (depending on the size of the screen).

I can remove `position: absolute` and with the following changes get the same result:

```scss
// _common.scss
.App > .ant-layout > .ant-layout-content {
  width: 70%;
  margin: 0 auto;
  ...
}
```

Is this better ?

Reply:
I noticed later that the navbar gets position fixed on mobile so the margin-top will still be needed on mobile. I usually try to avoid position: absolute but here I think it's a matter of preference.

For the content I prefer to use `max-width: $container-width; width: 100%; padding: 0 $gutter;` and avoid having multiple widths depending on media queries.